js/image/shared/Webcam.svelte

```diff
@@ -253,6 +282,10 @@
 		height: var(--size-full);
 	}
 
+	.invisible {
```

Comment:
nit, but we use the class name `hide` throughout the codebase for `display: none;`, for consistency would be nice